platform-sdk/docs/base/logging/logging.md

```diff
@@ -0,0 +1,177 @@
+## Hedera Logging API
+
+### Introduction
+The Hedera Logging API is a custom logging facade which is crafted to meet our unique requirements, ensuring compatibility with various software integrations. It acts as a bridge, redirecting events from our facade to SLF4J, providing flexibility for projects dependent on our modules.
+
+### Architecture and Configuration
+The architecture does not dictate the format of logging output, allowing for a wide range of output options. This flexibility permits various configurations, from console outputs to integrations with systems like Graylog or Kibana.
+
+### Features of the Logging Facade
+The facade goes beyond mere message logging, encapsulating rich information within each logging event. Key features include:
+
+1. **Support for Message Placeholders**
+2. **Event Origin Tracking**
+3. **Timestamp Inclusion**
+4. **Thread Identification**
+5. **Throwable Cause Handling**
+6. **Support for Various Log Levels**
+7. **Marker Support**
+8. **Key-Value Based Metadata**
+9. **Mapped Diagnostic Context (MDC) Support**
+
+---
+
+## Using the Hedera Logging API
+
+### Setting Up the Logger
+
+To integrate the new logging API within your Java application, begin by establishing a logger instance in your class. Here’s how you can set it up:
+
+1. **Import the Logger Class**:
+   Start by importing the `Loggers` class from the `com.swirlds.logging.api` package.
+
+   ```java
+   import com.swirlds.logging.api.Loggers;
+   ```
+
+2. **Create a Logger Instance**:
+   In your class, declare a `private static final` logger instance. Use the `Loggers.getLogger()` method, passing your class (`MyClass.class`) as a parameter.
+
+   ```java
+   public class MyClass {
+       private static final Logger LOGGER = Loggers.getLogger(MyClass.class);
```

Comment:
As far as I know we defined to use `logger` instead of `LOGGER`. Pls double check our coding guidelines.